src/bin/cargo-fmt.rs

```diff
@@ -288,16 +288,16 @@ fn format_files(files: &[PathBuf],
         println!("");
     }
     let mut command = try!(Command::new("rustfmt")
-        .stdout(stdout)
-        .args(files)
-        .args(fmt_args)
-        .spawn()
-        .map_err(|e| match e.kind() {
-            std::io::ErrorKind::NotFound => {
-                std::io::Error::new(std::io::ErrorKind::Other,
-                                    "Could not run rustfmt, please make sure it is in your PATH.")
-            }
-            _ => e,
-        }));
+                               .stdout(stdout)
```

Comment:
I don't understand your suggestion for a fix - it seems both the fixed code and the newly broken code only have a single argument and I don't know what `arg_is_block` means.